libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/accounts/MapboxNavigationAccounts.kt

```diff
@@ -71,6 +73,16 @@ class MapboxNavigationAccounts private constructor() : SkuTokenProvider {
         } ?: resourceUrl
     }
 
+    /**
+     * Returns current SDK SKU token needed for ART.
+     */
+    override fun obtainSkuToken(): String {
+        return skuGenerator?.let { generator ->
```

Comment:
maybe :)

return skuGenerator?.also { generateToken() } ?: EMPTY_SKU

Reply:
The compiler doesn't like it 😬 

![obtainSkuToken suggestion](https://user-images.githubusercontent.com/1668582/81844122-9afb3a00-951c-11ea-83a4-aa3921aa131f.png)

Reply:
oh yeah, also and apply return SkuGenerator

you can just 
```
return skuGenerator?.generateToken() ?: EMPTY_SKU
```

Reply:
Even better

```kotlin
override fun obtainSkuToken(): String = skuGenerator?.generateToken() ?: EMPTY_SKU
```

Thanks for the suggestion!